ansible/roles/ocp4_workload_rhacm/readme.adoc

```diff
@@ -0,0 +1,124 @@
+*** Debug task will print out: `pre_workload Tasks completed successfully.`
+
+** Playbook: link:./tasks/workload.yml[workload.yml] - Used to deploy ACM
+*** This role deploys ACM (project)
+*** Debug task will print out: `workload Tasks completed successfully.`
+
+** Playbook: link:./tasks/post_workload.yml[post_workload.yml] - Used to
+ configure the workload after deployment
+*** This role doesn't do anything here
+*** Debug task will print out: `post_workload Tasks completed successfully.`
+
+** Playbook: link:./tasks/remove_workload.yml[remove_workload.yml] - Used to
+ delete the workload
+*** This role uninstalls ACM and related resources
+*** Debug task will print out: `remove_workload Tasks completed successfully.`
+
+== Review the defaults variable file
+
+* This file link:./defaults/main.yml[./defaults/main.yml] contains all the variables you need to define to control the deployment of your workload.
+* The variable *ocp_username* is mandatory to assign the workload to the correct OpenShift user.
+* A variable *silent=True* can be passed to suppress debug messages.
+* Other variables:
+** *ocp4_workload_rhacm_acm_project*: The name of the project where ACM is deployed. Default: `open-cluster-management`
+** *ocp4_workload_rhacm_acm_release*: ACM release to be deployed. Default: `release-1.0`
+** *ocp4_workload_rhacm_docker_registry*: ACM Container Registry. Default: `registry.access.redhat.com/rhacm1-tech-preview`
+
+* You can modify any of these default values by adding `-e "variable_name=variable_value"` to the command line
+
+=== Deploy a Workload with the `ocp-workload` playbook [Mostly for testing]
+
+----
+TARGET_HOST="bastion.na311.openshift.opentlc.com"
+OCP_USERNAME="mavazque-redhat.com"
+WORKLOAD="ocp4_workload_rhacm"
+GUID=1001
+REGISTRY_USER="<user>"
+REGISTRY_PASSWORD="<password>"
+
+# a TARGET_HOST is specified in the command line, without using an inventory file
+ansible-playbook -i ${TARGET_HOST}, ./configs/ocp-workloads/ocp-workload.yml \
+    -e"ansible_ssh_private_key_file=~/.ssh/keytoyourhost.pem" \
+    -e"ansible_user=ec2-user" \
+    -e"ocp_username=${OCP_USERNAME}" \
+    -e"ocp_workload=${WORKLOAD}" \
+    -e"ocp4_workload_rhacm_docker_username=${REGISTRY_USER}" \
+    -e"ocp4_workload_rhacm_docker_password=${REGISTRY_PASSWORD}" \
+    -e"silent=False" \
+    -e"guid=${GUID}" \
+    -e"ACTION=create"
+----
+
+=== To Delete an environment
+
+----
+TARGET_HOST="bastion.na311.openshift.opentlc.com"
+OCP_USERNAME="mavazque-redhat.com"
+WORKLOAD="ocp4_workload_rhacm"
+GUID=1002
+
+# a TARGET_HOST is specified in the command line, without using an inventory file
+ansible-playbook -i ${TARGET_HOST}, ./configs/ocp-workloads/ocp-workload.yml \
+    -e"ansible_ssh_private_key_file=~/.ssh/keytoyourhost.pem" \
+    -e"ansible_user=ec2-user" \
+    -e"ocp_username=${OCP_USERNAME}" \
+    -e"ocp_workload=${WORKLOAD}" \
+    -e"guid=${GUID}" \
+    -e"ACTION=remove"
+----
+
```

Comment:
I usually delete everything after that line. It's just not helpful...

Reply:
done.